mslib/msui/qt5/ui_mscolab_window.py

```diff
@@ -254,12 +251,10 @@ def retranslateUi(self, MSSMscolabWindow):
         _translate = QtCore.QCoreApplication.translate
         MSSMscolabWindow.setWindowTitle(_translate("MSSMscolabWindow", "Mission Support Collaboration"))
         self.label_2.setText(_translate("MSSMscolabWindow", "MSColab URL:"))
-        self.connectMscolab.setToolTip(_translate("MSSMscolabWindow", "Connect to Mscolab server"))
-        self.connectMscolab.setWhatsThis(_translate("MSSMscolabWindow", "Connect to mscolab server"))
-        self.connectMscolab.setText(_translate("MSSMscolabWindow", "Connect"))
+        self.toggleConnectionBtn.setToolTip(_translate("MSSMscolabWindow", "Connect/Disconnect to Mscolab server"))
```

Comment:
there is different text in the ui file

Reply:
Since a single button is handling both connect and disconnect, I had changed the text a bit. I removed the old disconnect button from qt designer and regenerated the py file so that it gets properly removed. Is that alright?

Reply:
@ReimarBauer

Reply:
That there the string is changed is correct. But it is not the same as when we generate this file from the ui file.
This file needs to be created by pyuic5 from the ui_mscolab_window.ui file. It looks like an older state.
Also the layout change is not shown in the diff.